loadgen/bindings/python_api.cc

```diff
@@ -220,7 +220,9 @@ PYBIND11_MODULE(mlperf_loadgen, m) {
       .def_readwrite("qsl_rng_seed", &TestSettings::qsl_rng_seed)
       .def_readwrite("sample_index_rng_seed",
                      &TestSettings::sample_index_rng_seed)
-      .def_readwrite("schedule_rng_seed", &TestSettings::schedule_rng_seed);
+      .def_readwrite("schedule_rng_seed", &TestSettings::schedule_rng_seed)
+      .def_readwrite("latency_percentile_multistream", &TestSettings::latency_percentile_multistream)
```

Comment:
To be consistent:
latency_percentile_multistream -> multi_stream_target_latency_percentile
and:
latency_percentile_server -> server_target_latency_percentile